src/android/app/src/main/java/org/citra/citra_emu/viewmodel/CompressProgressDialogViewModel.kt

```diff
@@ -0,0 +1,33 @@
+// Copyright Citra Emulator Project / Azahar Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+package org.citra.citra_emu.viewmodel
+
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+
+object CompressProgressDialogViewModel: ViewModel() {
+    private val _progress = MutableStateFlow(0)
```

Comment:
I don't personally mind one way or the other, but I don't think we use the `_varName` pattern for private fields anywhere else I have seen.